subsys/random/Kconfig

```diff
@@ -75,6 +75,19 @@ config XOSHIRO_RANDOM_GENERATOR
 
 endchoice # RNG_GENERATOR_CHOICE
 
+
+DT_CHOSEN_Z_ENTROPY := zephyr,entropy
+config CSPRNG_AVAILABLE
+	bool
+	default y if $(dt_chosen_enabled,$(DT_CHOSEN_Z_ENTROPY))
+	help
+	  Helper that can be used to check if the platform is capable of generating
+	  CS random values. For this to be enabled, there must be the "zephyr,entropy"
```

Comment:
Complete nitpick:
CS acronym is introduced only below in the file. So I'd make it explicit in this help section.